org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/javaeditor/JavaElementImplementationHyperlink.java

```diff
@@ -269,15 +269,13 @@ public void acceptSearchMatch(SearchMatch match) throws CoreException {
 				}
 				try {
 					String typeLabel= JavaElementLabels.getElementLabel(type, JavaElementLabels.DEFAULT_QUALIFIED);
-					monitor.beginTask(Messages.format(JavaEditorMessages.JavaElementImplementationHyperlink_search_method_implementors, typeLabel), 10);
+					monitor.setTaskName(Messages.format(JavaEditorMessages.JavaElementImplementationHyperlink_search_method_implementors, typeLabel));
```

Comment:
Do you actually **see** the task name on the status line?
I don't see it neither with nor without this patch.

Reply:
i don't see it either